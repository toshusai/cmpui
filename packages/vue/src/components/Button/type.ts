export const ButtonVariant = {
  primary: "primary",
  secondary: "secondary",
} as const;

export type ButtonVariant = (typeof ButtonVariant)[keyof typeof ButtonVariant];

export const ButtonSize = {
  S: "S",
  M: "M",
  L: "L",
} as const;

export type ButtonSize = (typeof ButtonSize)[keyof typeof ButtonSize];

export type ButtonProps = {
  variant: ButtonVariant;
  size: ButtonSize;
  disabled: boolean;
};
