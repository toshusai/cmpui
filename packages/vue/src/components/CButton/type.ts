export const CButtonVariant = {
  primary: "primary",
  secondary: "secondary",
} as const;

export type CButtonVariant =
  (typeof CButtonVariant)[keyof typeof CButtonVariant];

export const CButtonSize = {
  S: "S",
  M: "M",
  L: "L",
} as const;

export type CButtonSize = (typeof CButtonSize)[keyof typeof CButtonSize];

export type CButtonProps = {
  variant?: CButtonVariant;
  size?: CButtonSize;
  disabled?: boolean;
};
