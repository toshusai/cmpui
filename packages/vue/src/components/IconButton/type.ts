export const IconButtonSize = {
  S: "S",
  M: "M",
} as const;

export type IconButtonSize =
  (typeof IconButtonSize)[keyof typeof IconButtonSize];

export type IconButtonProps = {
  bordered?: boolean;
  disabled?: boolean;
  rounded?: boolean;
  selected?: boolean;
  size?: IconButtonSize;
};
