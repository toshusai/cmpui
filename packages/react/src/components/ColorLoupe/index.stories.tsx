import type { Meta, StoryObj } from "@storybook/react";

import { ColorLoupe } from ".";

const meta: Meta<typeof ColorLoupe> = {
  title: "Colors/ColorLoupe",
  component: ColorLoupe,
};

export default meta;
type Story = StoryObj<typeof ColorLoupe>;

export const Default: Story = {
  render: function Render() {
    return (
      <div
        style={{
          position: "relative",
          width: "100px",
          height: "100px",
        }}
      >
        <ColorLoupe color="#c649ba" x={50} y={50} />
      </div>
    );
  },
};
