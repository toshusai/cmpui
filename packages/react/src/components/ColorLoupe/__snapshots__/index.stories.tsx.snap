// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors/ColorLoupe Default smoke-test 1`] = `
<div style="position: relative; width: 100px; height: 100px;">
  <svg width="1"
       height="25"
       viewbox="0 0 1 25"
       style="position: absolute; overflow: visible; pointer-events: none; left: 49.5px; top: 17.5px;"
  >
    <line stroke="var(--cmpui-border-color)"
          stroke-width="1"
          x1="0.5"
          x2="0.5"
          y1="24.5"
          y2="0.5"
    >
    </line>
  </svg>
  <div x="50"
       y="18"
       class="cmpui_circle__root alpha-checker-board"
       style="--cmpui-circle-nob-fill: white; --cmpui-circle-nob-radius: 16px; --cmpui-circle-nob-stroke-color: var(--cmpui-border-color); --cmpui-circle-nob-stroke-width: 1px; border: 1px solid var(--cmpui-border-color); left: 34px; top: 2px; z-index: 1; --block-size: 4px;"
  >
  </div>
  <div x="50"
       y="18"
       class="cmpui_circle__root"
       style="--cmpui-circle-nob-fill: #c649ba; --cmpui-circle-nob-radius: 16px; --cmpui-circle-nob-stroke-color: var(--cmpui-border-color); --cmpui-circle-nob-stroke-width: 1px; border: 1px solid var(--cmpui-border-color); left: 34px; top: 2px; z-index: 1;"
  >
  </div>
</div>
`;
